azure/secrets/template.json

```diff
@@ -11,6 +11,8 @@
     }
   },
   "variables": {
+    "principal_Microsoft_Azure_App_Service": "a6621090-e704-45ec-b65f-50257f9d4dcd",
```

Comment:
Just wanted to double check you intended to hardcode this UUID

Reply:
I did. The resource principals are constant for all environments inside a specific tenant. That is, every DfE subscription shares the same ID.

I've used `_`s as a delimeter and in place of spaces in the name of the principal, since the easiest way to look up these IDs is by manually adding them to a Key Vault's access policies and looking at the template export.